### PR TITLE
[MIG][17.0] purchase_default_terms_conditions

### PR DESCRIPTION
**Pull request by aisopuro**
This builds on the stale PR previously opened for this: #2180

### CODE REVIEW

**Comment by aisopuro** [comment]
There are some pre-commit fixing commits that technically could be squashed together, but I didn't want to do that to retain original author information.

File: purchase_default_terms_conditions/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "name": "Purchase Default Terms Conditions",
     "summary": """
         This module allows purchase default terms & conditions""",
-    "version": "16.0.1.0.2",
+    "version": "17.0.1.0.2",
```
Comment (amkarthik) [inline purchase_default_terms_conditions/__manifest__.py]: @nabilragab Please bump version to 17.0.1.0.0

Comment (aisopuro) [reply purchase_default_terms_conditions/__manifest__.py]: I squashed this into @nabilragab 's original migration commit, should be fixed now @amkarthik

**Comment by HaraldPanten** [comment]
Superseded by https://github.com/OCA/purchase-workflow/pull/2461 because:

- Several pre-commit fixes in the same migration.
- Pre-commit automatic fixes and manual pre-commit fixes (it's not the usual workflow).
- No activity since more than one month ago

**Pull request closed without merging**